.ci/common.sh

```diff
@@ -51,20 +51,8 @@ check_core_dumps() {
 }
 
 setup_deps() {
-	sudo pip install --upgrade pip
-	sudo pip install neovim
-
-	# For pip3
-	# https://github.com/travis-ci/travis-ci/issues/1528
-	# sudo apt-get install -q python3.3-dev
-	# curl -Ss http://python-distribute.org/distribute_setup.py | sudo python3
-	# curl -Ss https://raw.github.com/pypa/pip/master/contrib/get-pip.py | sudo python3
-	# sudo pip3.3 install neovim
-
 	if [ "$BUILD_NVIM_DEPS" != "true" ]; then
```

Comment:
To get the deps on OS X, you would need to add a condition on `$TRAVIS_OS_NAME` here and put back the `setup_deps` calls into the build files.